common/defs/bitcoin/faircoin.json

```diff
@@ -4,7 +4,7 @@
   "coin_label": "FairCoin",
   "website": "https://www.faircoin.world",
   "github": "https://github.com/faircoin/faircoin",
-  "maintainer": "Santi Noreña <santi@criptomart.net>",
+  "maintainer": "Santi Nore\u00f1a <santi@criptomart.net>",
```

Comment:
Please, let's not change this.

Reply:
Oops, sorry for the mistake... 
Fixed and force-pushed https://github.com/trezor/trezor-firmware/pull/399/commits/1528853c4de0c964294c74d28919e4a2b4c4f3bf.